src/rt-app_parse_config.c

```diff
@@ -785,6 +785,30 @@ static void parse_cpuset_data(struct json_object *obj, cpuset_data_t *data)
 	log_info(PIN "key: cpus %s", data->cpuset_str);
 }
 
+static void parse_numa_data(struct json_object *obj, numaset_data_t *data)
+{
+	data->numaset = NULL;
+	data->numaset_str = get_string_value_from(obj, "nodes", TRUE, NULL);
```

Comment:
So the data is always new?  Do we ever need to free the previous value prior to setting the new pointer?

Reply:
This function is called once for each configured task at init time.
It would be nice to have general free mechanism, but they do not have it :)